### PR TITLE
fix(BBD-880): Search page doesn't load on refresh

### PR DESCRIPTION
**Pull request by Filip-F**
https://github.com/groupby/flux-capacitor/pull/147
https://github.com/groupby/storefront-presets/pull/54
https://github.com/groupby/storefront-products/pull/30
https://github.com/groupby/storefront-paging/pull/18
https://github.com/groupby/storefront-breadcrumbs/pull/22
https://github.com/groupby/storefront-navigation/pull/34
https://github.com/groupby/storefront-did-you-mean/pull/17
https://github.com/groupby/storefront-related-queries/pull/17
https://github.com/groupby/storefront-record-count/pull/20

### CODE REVIEW

File: test/unit/record-count.ts [hunk 1]
```diff
@@ -26,14 +26,18 @@ suite('RecordCount', ({ expect, stub, spy, itShouldBeConfigurable, itShouldHaveA
...
+          total: QUERY,
+        });
       });
     });
   });
 
   describe('init()', () => {
     it('should listen for events', () => {
       const on = spy();
+      recordCount.updatePageRange = spy();
```
Comment (Filip-F) [inline test/unit/record-count.ts]: No need to be a spy.

File: src/record-count/index.ts [hunk 2]
```diff
@@ -11,10 +11,12 @@ class RecordCount {
     }
   };
   state: RecordCount.State = {
+    total: this.select(Selectors.recordCount),
     query: this.select(Selectors.currentQuery)
   };
 
   init() {
+    this.updatePageRange(this.select(Selectors.pageObject));
```
Comment (treeforever) [inline src/record-count/index.ts]: only page range is affected by racing? Not record count or query?

Comment (Filip-F) [reply src/record-count/index.ts]: Done

Comment (Vic-Dev) [reply src/record-count/index.ts]: Isn't calling all three of these going to cause 3 updates? Should probably not do that.

Comment (zfcb) [reply src/record-count/index.ts]: Oh actually the original code was correct, record count and query were both set in the constructor and thus we only need to update page range.

Comment (treeforever) [reply src/record-count/index.ts]: Then why don't you do this in the same place as on line 14 and 15?
In some other components, state from flux are initiated in a constructor. I don't really know its difference from doing that and doing this.

**Pull request merged**